query-engine/core/src/query_graph_builder/read/utils.rs

```diff
@@ -259,6 +260,13 @@ pub(crate) fn get_relation_load_strategy(
     nested_queries: &[ReadQuery],
     query_schema: &QuerySchema,
 ) -> QueryGraphBuilderResult<RelationLoadStrategy> {
+    static RELATION_LOAD_STRATEGY: Lazy<Result<RelationLoadStrategy, DomainError>> =
+        Lazy::new(|| std::env::var("RELATION_LOAD_STRATEGY").unwrap().as_str().try_into());
```

Comment:
(conversely, `unwrap`ping the result of `try_into` would've actually been fine and maybe even preferable if you want to make an invalid value an error instead of silently ignoring it)